src/dotnet-interactive/ContentFiles/kernels/.net-csharp/kernel.json

```diff
@@ -2,6 +2,7 @@
   "argv": [
     "dotnet",
     "interactive",
+    "[jupyter]",
```

Comment:
This should be unnecessary, since we already know the frontend based on the verb.

Reply:
`dotnet interactive jupyter install` will have jupyter as verb but no fronted while the kernel spec change is explicit about it.

Maybe processing that verb to decide about the production of a frontend proeprty for the juptyer verb when is not a jupyter install gesture?

Reply:
> Maybe processing that verb to decide about the production of a frontend proeprty for the juptyer verb when is not a jupyter install gesture?

That should work, no?

Reply:
will do that then